src/marqo/tensor_search/tensor_search.py

```diff
@@ -2692,7 +2690,7 @@ def delete_documents(config: Config, index_name: str, doc_ids: List[str]):
         )
     )
 
-def normalize_vector(embeddings: Union[List[List[float]], ndarray]) -> List[List[float]]:
+def normalize_vector(embeddings: Union[List[List[float]], ndarray, List[float]]) -> List[List[float]]:
```

Comment:
What use case uses a List[float]?

Reply:
None as of now, this change was based off of Joshua's comment here: https://github.com/marqo-ai/marqo/pull/970 that there might be a future need to accept List[float] type

Reply:
@vicilliar , could you clarify why we need List[float] here?

Reply:
This was closed during standup.